docs/installing.rst

```diff
@@ -0,0 +1,20 @@
+Installation
+============
+
+Requirements
+------------
+FoxPuppet will work with python 2.6, 2.7, 3.3, 3.4, 3.5, and 3.6
```

Comment:
We don't test 3.6 yet :) I would just list 2.7 and 3.5 for now as that's what we test via Travis CI.